### ISSUE TITLE
FEATURE Request (not an issue)

### ISSUE THREAD

**Issue by ruchirarya** [jschicht/RawCopy#20]
First of all thanks for making this great tool.

This would have been a truly amazing tool for the DFIR community if the tool supported wildcards.
For example, it would have been awesome to copy all SAM* files with just one command:
"c:\windows\system32\config\SAM*"

**Comment by undrline** [comment]
Yes, supporting wildcards would be great.  Also other functions along those lines: exclude files/directories, limit to just files or just directories, etc
